lib/spark.js

```diff
@@ -100,7 +99,8 @@ export default class Spark {
         controller.abort()
       }, 60_000)
     }
-    maxDurationTimeout = setTimeout(() => {
+
+    const maxDurationTimeout = setTimeout(() => {
```

Comment:
thanks for this clean up!

Reply:
You are welcome. It was reported by `npx standard`. I am surprised this passed our CI checks 🤔

Reply:
https://github.com/CheckerNetwork/spark-checker/pull/127#discussion_r2018162510